bitwarden_license/src/Scim/Utilities/ExceptionHandlerFilterAttribute.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Bit.Core.Exceptions;
+using Bit.Scim.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Bit.Scim.Utilities;
+
+public class ExceptionHandlerFilterAttribute : ExceptionFilterAttribute
```

Comment:
Is this meaningfully different than the sharedutils one? Can we use that one instead, or do we need a new version?

Reply:
I created this to return a `ScimErrorResponseModel` in the response body.

Reply:
@MGibson1 just following up on this. To expand on @r-tome's explanation, the SCIM specification requires a certain response format (which we represent with `ScimErrorResponseModel`), this lets us handle thrown exceptions according to SCIM requirements rather than our general app requirements.